### PR TITLE
[GLOSSARY] proof of work

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds pow to glossary

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ The scriptPubKey is also modified.
 Read more: [BIP 84 derivation scheme for P2WPKH based accounts](/using-wasabi/BIPs.md#bip-84-derivation-scheme-for-p2wpkh-based-accounts)
 :::
 
+:::details
+### Proof of Work (POW)
+
+A Bitcoin block is only valid if the hash of it is below a certain difficulty target.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: A Bitcoin block is only valid if its hash is below a certain difficulty target.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
A Bitcoin block is only valid if its hash is below a certain difficulty target.
```

FYI, you can suggest an edit which I can easily commit by clicking this button, and then editing the text.

screenshot_20191222_190228

Comment (SimoneBovi) [reply docs/glossary/Glossary.md]: Sorry Max, I didn't know about that. Too late, I guess :) Thanks!

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -96,6 +96,14 @@ The scriptPubKey is also modified.
 Read more: [BIP 84 derivation scheme for P2WPKH based accounts](/using-wasabi/BIPs.md#bip-84-derivation-scheme-for-p2wpkh-based-accounts)
 :::
 
+:::details
+### Proof of Work (POW)
+
+A Bitcoin block is only valid if its hash is below a certain difficulty target.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
One of the requirements for a Bitcoin block to be valid is its hash should be below a certain difficulty target.
```

File: docs/glossary/Glossary.md [hunk 3]
```diff
@@ -96,6 +96,14 @@ The scriptPubKey is also modified.
...
+In order to create a valid block, a miner must thus repeatedly hash a candidate block with changing nonce until by sheer luck he finds a hash with low value.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
In order to create a valid block, a miner must repeatedly hash a candidate block with a changing nonce until by sheer luck he finds a hash with low value.
```

File: docs/glossary/Glossary.md [hunk 4]
```diff
@@ -96,6 +96,14 @@ The scriptPubKey is also modified.
...
+By providing this pre-image block and hash, anyone can verify the amount of computational energy that a miner on average should have invested in order to find this block.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
By providing this pre-image block, anyone can verify the amount of computational energy that a miner on average should have invested in order to find this block.
```

**Comment by yahiheb** [comment]
As @RiccardoMasutti noted in some PRs, I am not sure if Bitcoin basics should be in the Wasabi glossary, but I don't have a strong opinion about that.

**Comment by MaxHillebrand** [comment]
PoW is mentioned [two](https://github.com/zkSNACKs/WasabiDoc/blob/c953e92b501b38df944fa876adc2d6388685a5b4/docs/FAQ/FAQ-UseWasabi.md#L190) [times](https://github.com/zkSNACKs/WasabiDoc/blob/c953e92b501b38df944fa876adc2d6388685a5b4/docs/FAQ/FAQ-UseWasabi.md#L1138) in the docs, so I think it should be included.

File: docs/glossary/Glossary.md [hunk 5]
```diff
@@ -96,6 +96,14 @@ The scriptPubKey is also modified.
 Read more: [BIP 84 derivation scheme for P2WPKH based accounts](/using-wasabi/BIPs.md#bip-84-derivation-scheme-for-p2wpkh-based-accounts)
 :::
 
+:::details
+### Proof of Work (POW)
+
+One of the requirements for a Bitcoin block to be valid is its hash should be below a certain difficulty target.
+In order to create a valid block, a miner must repeatedly hash a candidate block with a changing nonce until by sheer luck he finds a hash with low value.
+By providing this pre-image block, anyone can verify the amount of computational energy that a miner on average should have invested in order to find this block.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]: Maybe we can add `and this is what is known as Proof of Work`

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
By providing this pre-image block, anyone can verify the amount of computational energy that a miner on average should have invested in order to find this block, and this is what is known as Proof of Work.
```

**Pull request merged**